odd-platform-ui/src/components/Management/TagsList/TagsList.tsx

```diff
@@ -148,7 +149,7 @@ const TagsListView: React.FC<TagsListProps> = ({
           </Typography>
         </Grid>
       </Grid>
-      {tagsList.length ? (
+      {tagsList.length && (
```

Comment:
With this condition the skeleton will be hidden until we get first page. I think it should be removed.